flipperdevices/flipperzero-firmware
# [FL-1153] iButton OTG pullup

# What's new
Enable OTG in one_wire_master

# Verification

- Build for f4 and f5 targets
- Change pull up resistor R18 on NFC board 1k -> 47k; Solder R69 1k resistor on MAIN board
- Verify correct iButton reading, writing and emulation

# Checklist (do not modify)
